operator/src/main/kubernetes/kubernetes.yml

```diff
@@ -72,12 +72,29 @@ rules:
       - delete
       - patch
       - update
+  - apiGroups:
+      - monitoring.coreos.com
+    resources:
+      - servicemonitors
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - update
+      - watch
```

Comment:
When using [raw K8s resources](https://github.com/keycloak/keycloak-k8s-resources/blob/nightly/kubernetes/kubernetes.yml) (instead of OLM), I assume K8s will also complain if `ServiceMonitor` CRD is not installed, right? Should we have a dedicated YAML with permissions for `ServiceMonitor`?

Reply:
> I assume K8s will also complain if `ServiceMonitor` CRD is not installed, right?

No that's not the case AFAIU. When you create a `ClusterRole/Role`, the only thing that is verified is the structure of the yaml. K8s does not ensure that the resource(s) are actually available on the system until you try to access them, at which point you will get a resource not found error before RBAC is verified.

You can test this by applying `kubernetes.yml` on a cluster that does not contain the ServiceMonitor CRD.